.github/ISSUE_TEMPLATE/account-recovery.yml

```diff
@@ -46,8 +47,15 @@ body:
     description: |
       By their nature account recovery issues are labor intensive. The PyPI team has limited resources available for their completion.
 
-      There may be a delay of a week or more between responses for these issues.
+      There may be a delay of few months or more between responses for these issues.
     options:
     - label: I understand that it may take a significant amount of time to process my account recovery request.
       required: true
+      
+- type: checkboxes
+  attributes:
+    label: Recovery code lost
+    options:
+    - label: I have lost my 2FA authentication app or security token, and my recovery code as well
+      required: false
 ...
```

Comment:
I have downloaded Microsoft Authenticator on my new cell phone and also, I lost my recovery codes, is there a way to set a new 2FA on my new cell phone? if not how can I solve this?
Thank you.